### PR TITLE
Update getting-started.adoc for windows

### PR DESCRIPTION
**Pull request by RotBolt**
Edit commands for cmd & powershell on windows
As discussed on #6452 with @gastaldi

### CODE REVIEW

File: docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -83,6 +85,22 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
 cd getting-started
 ----
 
+For windows users
```
Comment (gastaldi) [inline docs/src/main/asciidoc/getting-started.adoc]:
```suggestion
For Windows users
```

File: docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -73,6 +73,8 @@ The solution is located in the `getting-started` directory.
 
 The easiest way to create a new Quarkus project is to open a terminal and run the following command:
 
+For linux & mac
```
Comment (gastaldi) [inline docs/src/main/asciidoc/getting-started.adoc]:
```suggestion
For Linux and MacOS users
```

File: docs/src/main/asciidoc/getting-started.adoc [hunk 3]
```diff
@@ -83,6 +85,22 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
...
+For windows users
+
+- If using cmd , (don't use forward slash `\`)
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create -DprojectGroupId=org.acme -DprojectArtifactId=getting-started -DclassName="org.acme.quickstart.GreetingResource" -Dpath="/hello"
+----
+
+- If using Powershell , execute `-D` parameters in double quotes
```
Comment (gastaldi) [inline docs/src/main/asciidoc/getting-started.adoc]:
```suggestion
- If using Powershell , wrap `-D` parameters in double quotes
```

**Review by gastaldi — DISMISSED** [review]
Looks good.I added a few minor changes to look better :)

File: docs/src/main/asciidoc/getting-started.adoc (hunk 2 above)
Comment (gsmet) [reply docs/src/main/asciidoc/getting-started.adoc]: If we want to be pedantic, it should be `macOS` :)

File: docs/src/main/asciidoc/getting-started.adoc (hunk 3 above)
Comment (KaenDagger) [reply docs/src/main/asciidoc/getting-started.adoc]: done

File: docs/src/main/asciidoc/getting-started.adoc (hunk 2 above)
Comment (KaenDagger) [reply docs/src/main/asciidoc/getting-started.adoc]: I have done with cap "M" for macOS

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks for taking the time to improve the doc!

**Pull request merged**